pkg/sink/codec/debezium/codec.go

```diff
@@ -49,10 +50,17 @@ func (c *dbzCodec) writeDebeziumFieldValues(
 	var err error
 	colInfos := tableInfo.GetColInfosForRowChangedEvent()
 	writer.WriteObjectField(fieldName, func() {
+		rows := make([]any, len(tableInfo.Columns))
```

Comment:
what is rows for ?

Reply:
for the generated column. Remove it since the generated column doesn't output